sdcm/cluster.py

```diff
@@ -4126,6 +4113,31 @@ def node_setup(self, node: BaseNode, verbose: bool = False, timeout: int = 3600)
 
             self._scylla_post_install(node, install_scylla, devname)
 
+            # prepare and start saslauthd service
+            if self.params.get('prepare_saslauthd'):
```

Comment:
should be in separate method `prepare_and_start_saslauthd_service`

Reply:
Good idea, will fix.

But this kind of improvement should not block merging of URGENT pr.